Modified get involved page to the current version of EvalAI

Sanji515 opened the pull request:
**Minor changes related to css**

- Changed the link color from red to yellow and some font related css to match with the theme of current version

- Link to live demo: http://pr-136-evalai.surge.sh 

<br>

**<p align="center">Old UI Screenshot 1:**</p>


![Screenshot from 2019-06-24 16-32-42](https://user-images.githubusercontent.com/32524438/60020470-b6174e00-96ad-11e9-8433-d04a19f76931.png)


**<p align="center">Modified UI Screenshot 1:</p>**


![Screenshot from 2019-07-03 19-06-02](https://user-images.githubusercontent.com/32524438/60602300-2a01d680-9dd1-11e9-8338-1e5d36dd90ea.png)



**<p align="center">Modified UI Screenshot 2:</p>**


![Screenshot from 2019-06-25 21-30-59](https://user-images.githubusercontent.com/32524438/60114585-bee15000-9791-11e9-95cc-9dc51af3a6bf.png)

Sanji515 committed:
modified get involved page

Shekharrajak reviewed (commented):
I don't see much difference. Can you let us know what are the changes you did in current version of EvalAI? Make sure you write your changes in PR description properly.

lunayach committed:
Merge branch 'master' into get-involved-page

lunayach commented:
Hi, @Sanji515 could you respond to @Shekharrajak and add some tests before we merge this?

Sanji515 committed:
changed the color to yellow of the links

Sanji515 committed:
Merge branch 'get-involved-page' of https://github.com/Sanji515/EvalAI-ngx into get-involved-page

Sanji515 committed:
Merge branch 'master' into get-involved-page

Sanji515 commented:
> Make sure you write your changes in PR description properly

@Shekharrajak @lunayach Updated the PR description, please have a look

Sanji515 committed:
added the unit tests for the scrollToTop method of global service

Sanji515 committed:
Merge branch 'get-involved-page' of https://github.com/Sanji515/EvalAI-ngx into get-involved-page

Sanji515 commented:
> add some tests before we merge this

done, @lunayach please review

Sanji515 commented:
@lunayach @Shekharrajak please review

lunayach commented:
Very minor things, but better if we don't leave them for the last moment. 

 ![Screenshot from 2019-06-25 14-34-50](https://user-images.githubusercontent.com/21227893/60075217-5ffee500-9757-11e9-999f-4a6f4ef4bb46.png)

UL list having "Get Involved", "CloudCV Website" etc. seems misaligned. (IMO)

 ![Screenshot from 2019-06-25 14-34-30](https://user-images.githubusercontent.com/21227893/60075218-5ffee500-9757-11e9-9955-af25351abf8f.png)

Should we have a border separating the paragraph above and "Report issues"? Also little too much whitespaces around the centred text. (IMO)

lunayach commented on src/app/components/get-involved/get-involved.component.spec.ts:
```diff
@@ -35,11 +37,19 @@ describe('GetInvolvedComponent', () => {
+    globalServiceSpy = spyOn(globalService, 'scrollToTop');
     component = fixture.componentInstance;
+
+    expect(globalServiceSpy).not.toHaveBeenCalled();
```
I'm not sure if this is helpful.

Sanji515 replied:
hey @lunayach it's nothing wrong to have that but also it doesn't make any sense to have that, so will remove that

lunayach commented:
`Merging #136 into master will decrease coverage by 0.01%.`

@Sanji515 if merging the code does not increase the code-coverage, we should make sure that the coverage is not decreased at least.

Sanji515 commented:
@lunayach I think that's the only test case can be written for this static page, please let me know me if I'm missing something

lunayach commented:
Okay, @Sanji515. It's fine for this PR, but please take a note of it for future PR's.

Sanji515 commented:
@lunayach yeah I do check report but even after adding that one test the report hasn't updated, it's still the same.

Sanji515 committed:
added the border bottom to the section title and removed the extra spaces

Sanji515 commented:
> UL list having "Get Involved", "CloudCV Website" etc. seems misaligned

@lunayach hey I think footer is going to be change a/c to our current version, so there's no need to do that

> Should we have a border separating the paragraph above and "Report issues"? Also little too much whitespaces around the centred text

done and updated the PR description, please review

Sanji515 commented:
@lunayach please review

Sanji515 committed:
Merge branch 'master' into get-involved-page

Sanji515 commented:
@lunayach can you please take a look at this?

lunayach commented:
HI @Sanji515, by the border in the above comment, I meant horizontal line separating the section beginning with "Report issues" and the introductory text above it. I guess, you may have misunderstood and underlined all the headings like "Report issues", "Press" instead. Those may not be required and could be removed. (IMO)

RishabhJain2018 committed:
Merge branch 'master' into get-involved-page

Sanji515 committed:
removed the heading border-bottom and added a border just below the intro text and above the first heading

Sanji515 committed:
Merge branch 'master' of https://github.com/Cloud-CV/EvalAI-ngx into get-involved-page

Sanji515 committed:
Merge branch 'get-involved-page' of https://github.com/Sanji515/EvalAI-ngx into get-involved-page

Sanji515 commented:
@lunayach sorry that I misunderstood the things, I've removed the border bottom of headings and added a border above the first heading and below the intro text

Please review

lunayach commented on src/app/components/get-involved/get-involved.component.scss:
```diff
@@ -69,10 +77,15 @@
     div {
       .get-involved-section-title {
-        margin-top: 50px;
+        margin: 0px !important;
```
Was `!important` needed? (and in other places)

Sanji515 replied:
@lunayach yeah actually it is not needed here, removed from here. I've used `!important` in one other place also but there it is needed so I've not removed from there please have a look

lunayach replied:
Okay.

Sanji515 committed:
removed the unnecessary use important in get involved css file

lunayach reviewed (approved):
Looks good to me. LGTM.
